### PR TITLE
Tuning

### PR DESCRIPTION
**Pull request by mlists**
Might want to add the sliding window changes to this PR

### CODE REVIEW

**Comment by mlists** [comment]
implements #183

File: components/shooter.py [hunk 1]
```diff
@@ -36,8 +37,10 @@ def setup(self) -> None:
         self.outer_motor.setInverted(True)
         self.centre_motor.setInverted(False)
 
-        self.outer_motor.setNeutralMode(ctre.NeutralMode.Coast)
-        self.centre_motor.setNeutralMode(ctre.NeutralMode.Coast)
+        for motor in self.outer_motor, self.centre_motor:
+            motor.configVelocityMeasurementPeriod(1)
```
Comment (auscompgeek) [inline components/shooter.py:40]: Does this actually work? I thought this needs to be an enum.

Comment (mlists) [reply components/shooter.py]: Seems to

**Review by auscompgeek — APPROVED** [review]
lgtm

**Pull request merged**